### PR TITLE
Use explicit select() in compute()

### LINKED ISSUE #1754: SQLite: Unexpected quotes in column names after group_by(...) %>% summarize() %>% ungroup() %>% compute()
```
> src_memdb() %>% copy_to(iris)
> src_memdb() %>% tbl("iris") %>%
  group_by(Species) %>% summarize() %>% ungroup %>%
  compute %>% colnames
[1] "\"Species\""
```

Is solved by explicitly selecting the column.

```
> src_memdb() %>% tbl("iris") %>%
  group_by(Species) %>% summarize() %>% ungroup %>%
  compute %>% select(Species) %>% colnames
[1] "Species"
```

Tested with 053a996cb12.

### PR DESCRIPTION
**Pull request by krlmlr**
To work around strange SQLite quoting behavior.

Fixes #1754.

### CODE REVIEW

**Comment by krlmlr** [comment]
@hadley: Could you please set GITHUB_PAT for dplyr on Travis?

File: R/tbl-sql.r [hunk 1]
```diff
@@ -375,7 +375,7 @@ compute.tbl_sql <- function(x, name = random_table_name(), temporary = TRUE,
   vars <- op_vars(x)
   assert_that(all(unlist(indexes) %in% vars))
   assert_that(all(unlist(unique_indexes) %in% vars))
-  db_save_query(x$src$con, sql_render(x), name = name, temporary = temporary)
+  db_save_query(x$src$con, sql_render(select_(x, .dots = vars)), name = name, temporary = temporary)
```
Comment (hadley) [inline R/tbl-sql.r]: Would be nice to have comment: "Forces explicit aliases of all variables, which avoids problems with SQLite quoting"

**Comment by krlmlr** [comment]
Comment + named variable.

**Comment by hadley** [comment]
Thanks!

**Pull request merged**